content/v2.0/views/views-and-exposures.md

```diff
@@ -108,3 +104,55 @@ module Bookshelf
   end
 end
 ```
+
+### Depending on other exposures
+
+Sometimes one exposure will depend on value of another. You can depend on another exposure by naming it as a positional argument for your exposure block.
```

Comment:
:name_badge: Might want to say *positional parameter* instead of *positional argument* since an argument is what you message the exposure with but when you define the exposure block itself, it's known as a parameter because it's part of the method definition.